### PR TITLE
Comment API GET and POST Request

### PR DESCRIPTION
**Pull request by araldortogul**
The Comment app provides support for GET and POST requests and a simple user interface.
The code for the app needs to be reviewed by 2 people before merging as discussed in meeting 15.

Some notes:
The timezone of practice_app is changed from "UTC" to "Europe/Istanbul".

### CODE REVIEW

File: practice_app/apps/user/templates/user-detail.html [hunk 1]
```diff
@@ -78,10 +78,10 @@ <h4 class="alert-heading">Failed</h4>
                             <input type="submit" name="submit" value="Get">
                             <br>
                     </form>
-                </div>	
+                </div>
             </div>
         </div>
 	</div>
     <a href="{%url 'user-home' %}"> <p style="text-align:center">Back to Home Page</p></a>
```
Comment (haticerk) [inline practice_app/apps/user/templates/user-detail.html:85]: As I understood, you also have to change this. Currently, user-api is not working properly.

Comment (haticerk) [reply practice_app/apps/user/templates/user-detail.html:85]: user-home to user:user-home

**Comment by Aliqaan** [comment]
Apart from what @haticerk mentioned, when I tried to run server after cloning the repo, creating a virtual environment and installing dependencies based on requirements.txt file it says crispy_forms module is not found. It might be a small mistake from earlier merges but it is better to fix it since we caught it. Can you add it to requirements.txt and commit that as well?

**Comment by araldortogul** [comment]
Thank you for your reviews @haticerk and @Aliqaan . I have updated **user-detail.html** and **requirements.txt** according to your comments.

**Review by haticerk — APPROVED** [review]

**Review by Aliqaan — APPROVED** [review]

**Pull request merged**